server/build_event_protocol/target_tracker/target_tracker.go

```diff
@@ -209,29 +209,20 @@ func (t *TargetTracker) writeTestTargets(ctx context.Context) error {
 	}
 	if len(updatedTargets) > 0 {
 		if err := updateTargets(ctx, t.env, updatedTargets); err != nil {
-			log.Warningf("Error updating targets: %s", err.Error())
+			log.Warningf("Error updating %s targets: %s", t.buildEventAccumulator.InvocationID(), err.Error())
```

Comment:
nit: you can use %q for the invocationID and it'll get quoted nicely

Reply:
(here and below a couple places too)

Reply:
Done